.devcontainer/devcontainer.json

```
@@ -0,0 +1,7 @@
+{
+  "image": "mcr.microsoft.com/devcontainers/universal:2",
```

Instead of using the `universal` image, wouldn't it be better to use the python dedicated one?

```suggestion
  "image": "mcr.microsoft.com/devcontainers/python:3",
```